src/types.ts

```diff
@@ -15,39 +16,4 @@ export interface ClientOption {
   endpoint: string;
 }
 
-export interface CreateTaskResponse {
-  id: string;
-}
-
-export interface ListTasksResponse {
-  tasks: {
-    id: string;
-    status: string;
-  }[];
-}
-
-export interface CancelTaskResponse {
-  value: boolean;
-}
-
-export interface DeleteTaskResponse {
-  value: boolean;
-}
-
-export interface SmartWallet {
-  address: string;
-  salt: string;
-  factory: string;
-}
-
-export interface CreateWalletReq {
-  salt: string;
-  factoryAddress?: string;
-}
-
-export interface TriggerMark {
-  blockNumber?: number;
-  logIndex?: number;
-  txHash?: string;
-  epoch?: number;
-}
```

Comment:
These code are not used any more since we refer directly to the variables in the protobuff.